source/cart.html

```diff
@@ -17,7 +17,11 @@ <h1 class="page-title has-dash">{{ page.name }}</h1>
             <div class="cart-item-quantity-holder" data-item-id="{{ item.id }}">
               {{ item | item_quantity_input, 'option-quantity', 'option-quantity' }}
             </div>
-            <a class="cart-item-remove" title="Remove item" href="#"><svg class="cart-remove-icon" width="11" height="11" xmlns="http://www.w3.org/2000/svg"><g fill-rule="evenodd"><path d="M10.11 1.596l-8.484 8.485-.707-.707L9.404.89z"/><path d="M1.626.89l8.485 8.484-.707.707L.919 1.596z"/></g></svg></a>
+            <a class="cart-item-remove" title="Remove item" href="#">
+              <svg width="11" height="11" xmlns="http://www.w3.org/2000/svg">

```

Comment:
Does this need `aria-hidden="true"` ?

Reply:
Hmm not sure. Can you remind me when that should be included for svgs?

Reply:
It seems to be decorative and the "Remove item" title exists so it's ok to hide it - @cella used the same technique in lunch break. More info here as well https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility

Reply:
Ah ok good to know!